ubtue_Brill.js

```diff
@@ -106,8 +106,8 @@ function extractErscheingunssjahr(date) {
 }
 
 function extractBerichtsjahr(dateEntry) {
-	let extractDate = dateEntry.split('(')[1].split(')')[0];
-		return extractDate;
+	let extractDate = dateEntry.match(/([^\)]\d{4})\):/)[1].replace('(', '');
```

Comment:
How about
```javascript
  let dateCandidate = dateEntry.match(/\(\s*(\d{4})\s*\)/);
  return dateCandidate.length > 1 ? dateCandidate[1] : null;
```

Reply:
yes, your suggestion is better approach. Your regex will match also the second occurence, if the input text is like so "Archiv für katholisches Kirchenrecht - Volume 186 (2019): Issue 2 (2021)". Therefore i will change it to `/\(\s*(\d{4})\s*\)/:?` to match pattern followed by colon`:`